crates/api_common/src/utils.rs

```diff
@@ -1162,6 +1162,19 @@ fn build_proxied_image_url(
   ))
 }
 
+pub fn check_code_verifier(code_verifier: &str) -> LemmyResult<&str> {
```

Comment:
Also this should probably be renamed to `check_pkce_code`

Reply:
This function is only used in a single file, so move it directly there and make it private.